server/database/build.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+DROP TABLE IF EXISTS users, tickets_status, tickets CASCADE;
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    email TEXT NOT NULL UNIQUE,
+    name VARCHAR NOT NULL ,
+    password VARCHAR NOT NULL
+);
+CREATE TABLE tickets_status(
+    uid SERIAL PRIMARY KEY,
+    status VARCHAR NOT NULL
+);
+CREATE TABLE tickets(
+    uid SERIAL PRIMARY KEY,
+    from_tickets VARCHAR NOT NULL,
+    to_tickets VARCHAR NOT NULL,
+    date INTEGER ,
+    body VARCHAR NOT NULL,
+    status  VARCHAR NOT NULL,
```

Comment:
@alaabadra Is this `status` different from the `ticket_status` table above?  Or does it reference the `ticket_status` table? In that case, I'd suggest using `REFERENCES`.

Reply:
we used it , because we considered status coming messages and status going messages